SMPyBandits/Policies/GreedyOracle.py

```diff
@@ -0,0 +1,34 @@
+from .IndexPolicy import IndexPolicy
+import numpy as np
+
+class GreedyPolicy(IndexPolicy):
+    """ Greedy Policy for rotting bandits.
+    Selects arm with best last value.
+    Reference: [Heidari , Kearns & Roth, 2016]
+    """
+    def __init__(self, nbArms):
+        super(GreedyPolicy, self).__init__(nbArms)
+        self.arms_history = [np.array([]) for arm in range(nbArms)]
+
+    def getReward(self, arm, reward):
+        super(GreedyOracle, self).getReward(arm, reward)
+        self.arms_history[arm] = np.insert(self.arms_history[arm], 0, 0) + reward
```

Comment:
This is a weird code! Just use `self.arms_history[arm].append(reward)` ?

Reply:
Indeed, all this array is not necessary. I never used this policy !